pkg/config/consts.go

```diff
@@ -43,13 +43,13 @@ const (
 	// ConsensusMaxStep consensus max step number.
 	ConsensusMaxStep = uint8(213)
 
-	// ConsensusMaxCommitteeSize represents the maximum size of the committee in
```

Comment:
As you're renaming this, why don't you use the new terminology -  `number of credits/number of slots` ?

Reply:
I was planning to do the renaming globally with a separate PR